var/spack/repos/builtin/packages/boost/package.py

```diff
@@ -155,6 +156,26 @@ def determine_toolset(self, spec):
         # fallback to gcc if no toolset found
         return 'gcc'
 
+    def bjam_python_line(self, spec):
+        from os.path import dirname, splitext
+        pydir = 'python%s.%s*' % spec['python'].version.version[:2]
```

Comment:
This would look more consistent with other Spack packages if it used:
```python
pydir = 'python' + spec['python'].version.up_to(2)
```
but not a big deal. Same below.